### DISCUSSION TITLE
https://anonymity-0.github.io/posts/portland/

### DISCUSSION THREAD

**Discussion by giscus** [Anonymity-0/Anonymity-0.github.io#3] [Announcements]
# https://anonymity-0.github.io/posts/portland/

PortLand: a scalable fault-tolerant layer 2 data center network fabric 💡 Meta Data Title
PortLand: a scalable fault-tolerant layer 2 data center network fabric
Journal
Authors
Radhika Niranjan Mysore; Andreas Pamboris; Nathan Farrington; Nelson Huang; Pardis Miri; Sivasankar Radhakrishnan; Vikram Subramanya; Amin Vahdat
Pub. date
八月 16, 2009
期刊标签
DOI
10.1145/1592568.1592575
附件
Niranjan Mysore 等 - 2009 - PortLand a scalable fault-tolerant layer 2 data c.pdf
📜 研究背景 &amp; 基础 &amp; 目的 这篇论文的研究背景是针对日益增长的数据中心网络中存在的限制和挑战，例如在现有的层2和层3网络协议在支持大规模数据中心中的灵活性、效率和容错性方面存在的局限性 在笔者提出的论文中，研究的基础是现有网络协议在面临未来单个站点拥有数百万虚拟终端的数据中心时所面临的挑战，特别是关注于如何在这种大规模环境中实现可扩展、易管理、容错和高效的数据中心网络结构

https://anonymity-0.github.io/posts/portland/

**Comment by Sylvialxy** [comment]
表扬

**Reply by Sylvialxy** [reply]
摇旗呐喊：加油！！！
